rssfeed/tests/requirements/110.txt

```diff
@@ -0,0 +1,4 @@
+django==1.10.1
```

Comment:
Might I suggest chaining requirements. It might make life a bit easier, when you need to handle even more version later on.

Just add a new requirements file in the directory call it something like `test_base.txt`.

Then just add `-r test_base.txt` in each requirement file. 
I am not 100% on how tox will pull it in, you might need to add the full path from project root to it, but i don't think so.

The base file can carry the requirements that are exactly the same between all version for instance `feedparser==5.2.1`